release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -0,0 +1,1538 @@
+
+|Cloning
+|Supported
+|Supported
+
+|Expansion
+|Supported
+|Supported
+
+|Snapshot
+|Supported
+|Supported
+|====
+
+[id="ocp-4-15-auth"]
+=== Authentication and authorization
+
+[id="ocp-4-15-networking"]
+=== Networking
+
+[id="ocp-4-15-registry"]
+=== Registry
+
+[id="ocp-4-15-storage"]
+=== Storage
+
+[id="ocp-4-15-oci"]
+=== Oracle&#174; Cloud Infrastructure
+
+[id="ocp-4-15-olm"]
+=== Operator lifecycle
+
+[id="ocp-4-15-osdk"]
+=== Operator development
+
+[id="ocp-4-15-builds"]
+=== Builds
+
+[id="ocp-4-15-machine-config-operator"]
+=== Machine Config Operator
+
+[id="ocp-4-15-machine-api"]
+=== Machine API
+
+[id="ocp-4-15-nodes"]
+=== Nodes
+
+[id="ocp-4-15-monitoring"]
+=== Monitoring
+
+The monitoring stack for this release includes the following new and modified features:
+
+[id="ocp-4-15-network-observability-1-5"]
+=== Network Observability Operator
+The Network Observability Operator releases updates independently from the {product-title} minor version release stream. Updates are available through a single, rolling stream which is supported on all currently supported versions of {product-title} 4. Information regarding new features, enhancements, and bug fixes for the Network Observability Operator is found in the xref:../network_observability/network-observability-operator-release-notes.adoc[Network Observability release notes].
+
+[id="ocp-4-15-scalability-and-performance"]
+=== Scalability and performance
+
+[id="ocp-4-15-hcp"]
+=== Hosted control planes (Technology Preview)
+
+
+[id="ocp-4-15-insights-operator"]
+=== Insights Operator
+
+[id="ocp-4-15-notable-technical-changes"]
+== Notable technical changes
+
+{product-title} {product-version} introduces the following notable technical changes.
+
+// Note: use [discrete] for these sub-headings.
+// example sub-heading below:
+//[discrete]
+//[id="ocp-4-15-cluster-cloud-controller-manager-operator"]
+//=== Cloud controller managers for additional cloud providers
+
+[id="ocp-4-15-deprecated-removed-features"]
+== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. For the most recent list of major functionality deprecated and removed within {product-title} {product-version}, refer to the table below. Additional details for more functionality that has been deprecated and removed are listed after the table.
+
+In the following tables, features are marked with the following statuses:
+
+* _General Availability_
+* _Deprecated_
+* _Removed_
+
+[discrete]
+[id="ocp-4-15-operators-dep-rem"]
+=== Operator lifecycle and development deprecated and removed features
+
+.Operator lifecycle and development deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.12 |4.13 |4.14
```

Comment:
```suggestion
|Feature |4.13 |4.14 |4.15
```

Reply:
And then all of the tables will need updated in the template to start the process.